swaggertooling/openapi-pipeline/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>swagger-tooling</artifactId>
+        <groupId>ch-swagger-tools</groupId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
I think this version should change from a snapshot @ch-dclarke do you agree?

Reply:
Yes, I completely agree. Since this is effectively releasing. We will need to check all the related POM files at the same time.

Reply:
Which POMs are those? everything under the Ch-spec project including render and specifications? Or leave those two out?

Reply:
I strongly suggest making it a separate Jira, and not tying it to the pipeline functionality.

Any of the POMs in ch-spec which reference snapshots, should reference versions in the future.